ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/EpochProcessorUtil.java

```diff
@@ -347,30 +349,41 @@ private static UnsignedLong get_base_reward(BeaconState state, int index) {
     }
 
     // Proposer and inclusion delay micro-rewards
-    for (Integer index : get_unslashed_attesting_indices(state, matching_source_attestations)) {
-      matching_source_attestations.stream()
-          .filter(
-              a ->
-                  get_attesting_indices(state, a.getData(), a.getAggregation_bits())
-                      .contains(index))
-          .min(Comparator.comparing(PendingAttestation::getInclusion_delay))
-          .ifPresent(
-              attestation -> {
-                UnsignedLong proposer_reward =
-                    get_base_reward(state, index)
-                        .dividedBy(UnsignedLong.valueOf(PROPOSER_REWARD_QUOTIENT));
-                rewards.set(
-                    attestation.getProposer_index().intValue(),
-                    rewards.get(attestation.getProposer_index().intValue()).plus(proposer_reward));
-                UnsignedLong max_attester_reward =
-                    get_base_reward(state, index).minus(proposer_reward);
-                rewards.set(
-                    index,
-                    rewards
-                        .get(index)
-                        .plus(max_attester_reward.dividedBy(attestation.getInclusion_delay())));
-              });
-    }
+
```

Comment:
(nit): might be a good idea to remove the empty line

Reply:
Done!